cli/minocontroller/actions.go

```diff
@@ -0,0 +1,102 @@
+// This file contains the implementation of the controller actions.
+//
+// Documentation Last Review: 07.10.2020
```

Comment:
Is this file copy-pasted from some other repo ?

Reply:
and if so, is there some reason we can't just use the default implementation of mino for this, if there's one ?

Reply:
Yes. And yes there is one. The mino controller from Dela would always connect to `127.0.01`, where here we are trying to connect to the public ip address.